src/CNRECS.DEF

```diff
@@ -1024,11 +1024,11 @@ RECORD SBCBASE "SBCBASE" *SUBSTRUCT *HIDEALL
   *r DBL sb_fSky		//   "view factor" to radiant surround at sky temp
   *r DBL sb_fAir		//   "view factor" to radiant surround at air temp (ground, horizon)
 
-  *s *e FLOAT sb_hcNat	// surface natural convection coefficient, Btuh/ft2-F
-  *s *e FLOAT sb_hcFrc	// surface forced (wind) convection coefficient, Btuh/ft2-F
+  *s *e FLOAT_GEZ sb_hcNat	// surface natural convection coefficient, Btuh/ft2-F
+  *s *e FLOAT_GEZ sb_hcFrc	// surface forced (wind) convection coefficient, Btuh/ft2-F
   *s *e FLOAT sb_hcMult	// surface convection coefficient multiplier, dimless
```

Comment:
`sb_hcMult` corresponds to the following surface convection coefficient multipliers:
`wnExHcMult`, `wnInHcMult`
`drExHcMult`, `drInHcMult`
`sfExHcMult`, `sfInHcMult`

`sb_hcMult` has been restricted to non-negative values, with corresponding changes to the documentation. Code changes were tested with negative and zero values of `drExHcMult`, `drInHcMult`, `sfExHcMult`, and `sfInHcMult`, resulting in the appropriate output. (`wnExHcMult`, `wnInHcMult` show the same behavior, though assignment of these data members has no effect; see below.)

Notes:
1. It appears to be impossible to assign `wnExHcMult` or `wnInHcMult`, as this would require `wnModel = Forward_Difference`, which is not allowed.

2. `sb_hxr` has also been restricted to non-negative values, but appears to be a calculated value only; it does not correspond to any data member.